hawkbit-ddi-resource/src/main/java/org/eclipse/hawkbit/ddi/rest/resource/DdiRootController.java

```diff
@@ -326,7 +344,10 @@ private ActionStatusCreate generateUpdateStatus(final DdiActionFeedback feedback
         }
 
         if (feedback.getStatus().getDetails() != null) {
-            messages.addAll(feedback.getStatus().getDetails());
+            messages.addAll(feedback.getStatus()
```

Comment:
Note: that query will page/limit and order based on the action status but I think that is ok. Important is that the DB takes care of the filtering.